content/en/continuous_integration/intelligent_test_runner/_index.md

```diff
@@ -161,6 +174,12 @@ The following environment variables must also be set:
 
 In addition to setting the environment variables above, you need to activate the Intelligent Test Runner on the [Test Service Settings][5] page.
 
+## Setup CI Job
+
+Intelligent Test Runner uses git metadata information (commit history) to work. However, some CI providers use a git shallow clone (`git clone --depth=0`) which only download the target commit without downloading any historical commit information. This setup does not contain enough information for Intelligent Test Runner. If your CI is using shallow clones, it must be changed.
```

Comment:
"This setup does not contain enough information for Intelligent Test Runner to work."